### PR TITLE
Updates to Notebook 1 & 2

### PR DESCRIPTION
**Pull request by kaitj**
* Convert notebooks 1 & 2 to carpentry format from Jupyter

### CODE REVIEW

File: _episodes/01-introduction_diffusion_data.md [hunk 1]
```diff
@@ -0,0 +1,282 @@
+---
+title: "Introduction to Diffusion"
+teaching: 20
+exercises: 5
+questions:
+- "How is dMRI data represented stored"
```
Comment (josephmje) [inline _episodes/01-introduction_diffusion_data.md:6]:
```suggestion
- "How is dMRI data represented"
```

File: _episodes/02-preprocessing.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+- "What are the standard preprocessing steps?"
+objectives:
+- "Understand the common preprocessing steps"
+keypoints:
+- "Many different preprocessing pipelines, dependent on how data is acquired"
+---
+
+## Diffusion Preprocessing
+
+Diffusion preprocessing typically comprises of a series of steps and is dependent on the data acquisition protocol. For the purposes of these lessons, the diffusion data has already been preprocessed and made available to use. Preprocessing is performed using a few well-known software packages (e.g. FSL, ANTS) and is chosen based on the acquisition protocol of the dataset <code>ds000030</code>.
```
Comment (josephmje) [inline _episodes/02-preprocessing.md:15]:
```suggestion
Diffusion preprocessing typically comprises a series of steps and is dependent on the data acquisition protocol. For the purposes of these lessons, the diffusion data has already been preprocessed and made available to use. Preprocessing is performed using a few well-known software packages (e.g. FSL, ANTS) and is chosen based on the acquisition protocol of the dataset <code>ds000030</code>.
```

File: _episodes/02-preprocessing.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+This dataset contains single shell data with one b0 image. This image is the volume of the diffusion series. There are 64 volumes with diffusion weighting.
+
+Here is an example preprocessing workflow from qsiprep:
+![preprocess](../fig/2/preprocess_steps.jpg)
+
+Like fMRI preprocessing, dMRI has some of the same challenges and some unique [ones](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3366862/).
+
+Our preprocesssing of this data will consist of two steps:
```
Comment (josephmje) [inline _episodes/02-preprocessing.md:24]:
```suggestion
Our preprocessing of this data will consist of two steps:
```

**Comment by josephmje** [comment]
looks good to me! just corrected a few typos.

**Comment by kaitj** [comment]
Looks good! Committed those corrections! 

Additional thought, should this repo be renamed to dMRI instead of DTI? I think we've both done that in our local copies anyways.

**Comment by josephmje** [comment]
> Looks good! Committed those corrections!
> 
> Additional thought, should this repo be renamed to dMRI instead of DTI? I think we've both done that in our local copies anyways.

Yes, I think it should. Just renamed!

**Pull request merged**